### PR TITLE
sanjuuni: init at 0.2

### PR DESCRIPTION
**Pull request by tomodachi94**
###### Description of changes
Add [Sanjuuni](https://github.com/MCJack123/sanjuuni) from v0.2 and add myself to the maintainers list.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by AndersonTorres — CHANGES_REQUESTED** [review]
Unfortunately the crappy interface of Github does not allow edit/comment  the commit messages. 

1. commit titles have only lowercase letters. There is no need to write `maintainers: Add My Self` when you can add `maintainers: add myself`
2. prefix version with `v`, `version`, `release`, `rel.`, `bumpy` or other strings in the commit title is not allowed even when the upstream project does it (and this is not even the case here).
3. you do not need to document each change and fixup like `fix sha256`, `fi typo` . Just rebase/squash them as if you made no error.
4. read the [Nipkgs manual](https://nixos.org/manual/nixpkgs/stable/#chap-submitting-changes)

**Comment by tomodachi94** [comment]
@AndersonTorres I *think* I've fixed the issues you've mentioned above.

File: pkgs/tools/graphics/sanjuuni/default.nix [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    install -Dm755 sanjuuni $out/bin/sanjuuni
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/MCJack123/sanjuuni";
+    description = "A command-line tool that converts images and videos into a format that can be displayed in ComputerCraft";
+    maintainers = [maintainers.tomodachi94];
```
Comment (AndersonTorres) [inline pkgs/tools/graphics/sanjuuni/default.nix:40]:
```suggestion
    maintainers = [ maintainers.tomodachi94 ];
```

File: pkgs/tools/graphics/sanjuuni/default.nix [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    license = licenses.gpl2;
```
Comment (AndersonTorres) [inline pkgs/tools/graphics/sanjuuni/default.nix:41]:
```suggestion
    license = licenses.gpl2Plus;
```

**Review by AndersonTorres — CHANGES_REQUESTED** [review]
Besides, LGTM

**Comment by tomodachi94** [comment]
@AndersonTorres done.

File: pkgs/tools/graphics/sanjuuni/default.nix [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    install -Dm755 sanjuuni $out/bin/sanjuuni
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/MCJack123/sanjuuni";
+    description = "A command-line tool that converts images and videos into a format that can be displayed in ComputerCraft";
+    maintainers = [ maintainers.tomodachi94 ];
+    license = licenses.gpl2Plus;
```
Comment (AndersonTorres) [inline pkgs/tools/graphics/sanjuuni/default.nix:41]:
```suggestion
    license = licenses.gpl2Plus;
    broken = stdenv.isDarwin;
```

Last thing:

As seen in the ofBorg messages, this software is failing in Darwin (MacOS) platforms.

So, it needs to add `broken = stdenv.isDarwin;` until we have a better understanding of this error.

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: pkgs/tools/graphics/sanjuuni/default.nix (hunk 3 above)
Comment (tomodachi94) [reply pkgs/tools/graphics/sanjuuni/default.nix:41]: Fixed in the latest push.

**Comment by tomodachi94** [comment]
Done.

File: pkgs/tools/graphics/sanjuuni/default.nix (hunk 3 above)
Comment (tomodachi94) [reply pkgs/tools/graphics/sanjuuni/default.nix:41]: Added that.

**Comment by AndersonTorres** [comment]
Next release remember to investigate the `broken`ness.

**Comment by AndersonTorres** [comment]
Why?

**Comment by tomodachi94** [comment]
> Why?

Are there any good reasons for not backporting a new package? I'm pretty new to this, but I believe there are no disadvantages (correct me if I'm wrong.)

**Comment by AndersonTorres** [comment]
You are asking for backport almost **two months** later - and after the end of year. 

```
bash$ git diff 131aff3f5e8ac211eac5094c0c78963121dd8cdd..upstream/release-22.11
   482081 1854757 23702872
bash$
```

Almost 500k-line diff.

Also, as I have seen, nothing about 

> investigate the brokenness

was vaguely tried.

I am very liberal about backporting changes, however you should be asked about this two months ago, the same day this was merged at least.

**Pull request merged**